parfive/downloader.py

```diff
@@ -376,6 +376,34 @@ def retry(self, results):
 
         return results
 
+    @classmethod
+    def quick_download(cls, urls, *, path=None, overwrite=None):
```

Comment:
The use of "quick" seems to imply that things will be downloaded faster though I don't have a better idea for the name of this function. `easy_download`? `download_wrapper`? None of those really sound better 😆

Reply:
`simple_download`?